tests/BUILD

```diff
@@ -146,7 +146,7 @@ jax_test(
 jax_test(
     name = "lobpcg_test",
     srcs = ["lobpcg_test.py"],
-    env = {"LOBPCG_EMIT_DEBUG_PLOTS": "1"},
+    env = {"LOBPCG_EMIT_DEBUG_PLOTS": "0"},
```

Comment:
I don't think we should change the default value here – these tests are run on existing CI processes and they're working as intended currently.

Reply:
Sure, we can stick with the previous version and add a required information in documentation